pkg/virt-controller/watch/vm/vm.go

```diff
@@ -130,6 +130,7 @@ func NewController(vmiInformer cache.SharedIndexInformer,
 	pvcInformer cache.SharedIndexInformer,
 	crInformer cache.SharedIndexInformer,
 	podInformer cache.SharedIndexInformer,
+	dvInformer cache.SharedIndexInformer,
```

Comment:
The informer is already passed above no? `dataVolumeInformer cache.SharedIndexInformer,`

Reply:
Done, thanks!